app/Transformers/TalkForConferenceTransformer.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace App\Transformers;
+
+use App\Conference;
+use App\Talk;
+
+class TalkForConferenceTransformer
+{
+    public static function transform(Talk $talk, Conference $conference)
+    {
+        /** @var Talk $currentTalk */
+        $currentTalk = $talk->current();
+
+        $submission = $talk->getMySubmissionForConference($conference);
+        $acceptance = $submission ? $submission->acceptance : null;
+
+        return [
+            'id' => $talk->id,
+            'title' => $currentTalk->title,
+            'url' => $currentTalk->getUrl(),
+            'submitted' => $acceptance ? false : !!$submission,
```

Comment:
Same note I gave here in the bad diff--I think I'd like to think of an accepted talk as still submitted as well.